### PR TITLE
Add Azure Stack HCI OS to List of Supported Operating Systems

### PR DESCRIPTION
**Pull request by steveharrington1**
### What does this PR do?
Adds Azure Stacj HCI to list of supported operating systems. It includes a link to the standard Windows agent install docs, as this same process works for Azure Stack HCI OS. It's basically just another flavor of Windows.

### Motivation
We want to participate in launch announcement with Microsoft, and one requirement is that we list support for it in our public docs.

### Preview
None.

### Additional Notes
None.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/basic_agent_usage/_index.md [hunk 1]
```diff
@@ -148,6 +148,7 @@ When the Agent is running, use the `datadog-agent launch-gui` command to open th
 | [macOS][9]                                      | macOS 10.12+                                      |
 | [Windows server 64-bit][10]                     | Windows Server 2008r2+ and Server Core (not Nano) |
 | [Windows 64-bit][10]                            | Windows 7+                                        |
+| [Windows Azure Stack HCI OS][10]                | All Versions                                      |
```
Comment (ruthnaebeck) [inline content/en/agent/basic_agent_usage/_index.md:151]: The link is pointing to: https://docs.datadoghq.com/agent/basic_agent_usage/windows/

Is that good?

Comment (steveharrington1) [reply content/en/agent/basic_agent_usage/_index.md:151]: Yes- the process is exactly the same as the instructions there indiciate. The only difference is that there is no GUI option, but the same cli/powershell commands that are already provided there work for AzSCHI. I don't think there's a need to call that out specifically anywhere though.

**Review by ruthnaebeck — APPROVED** [review]
👍

**Comment by steveharrington1** [comment]
@ruthnaebeck can you merge this? I am blocked from doing so. Thanks!

**Pull request merged**